dotcom-rendering/src/experiments/cypress-switches.ts

```diff
@@ -14,6 +14,7 @@ const cypressSwitches = {
 	abSignInGateMainControl: true,
 	abSignInGateMainVariant: true,
 	abSignInGateAusMandatory: true,
```

Comment:
We can delete this switch for AusMandatory I think.

Reply:
Ok, deleted